src/EnergyPlus/PollutionModule.cc

```diff
@@ -6488,6 +6493,20 @@ void GetFuelFactorInfo(EnergyPlusData &state,
                 fuelSourceFactor = 1.084 / Pollution.PurchHeatEffic;
             }
 
+        } else if (SELECT_CASE_var == "DistrictHeatingSteam") {
```

Comment:
Is `SELECT_CASE_var` on a `string` necessary? Don't we have this as an `enum` somewhere?